civihr_default_theme/scss/overrides/_datepicker-civicrm.scss

```diff
@@ -7,16 +7,11 @@
 // Override styles for civicrm datepicker specific to SSP
 .ui-datepicker {
   .ui-datepicker-calendar thead tr th {
-    font-size:13px !important;
+    font-size: 11.9px !important; //Calendar heading font size as in SSP
```

Comment:
Why `11.9px`? we never used it before. Cant you use 12px `instead`?

Reply:
Right `12px` is better. Will update.